src/components/features/DashboardView.tsx

```diff
@@ -0,0 +1,35 @@
+import { useGetTasksQuery } from "../../gql/graphql";
+import Error from "./../errors/Error";
+import TaskColumnSkeleton from "../ui/loadingSkeletons/TaskColumnSkeleton";
+import { useAppStore } from "../../store/store";
+import GridView from "./GridView";
+import TableView from "./TableView";
+import { statusOptions } from "../../libs/utils";
+
+const DashboardView = () => {
+  const viewMode = useAppStore((state) => state.viewMode);
+  const searchByTaskName = useAppStore((state) => state.searchByTaskName);
+
+  const { loading, error, data } = useGetTasksQuery({
+    variables: searchByTaskName !== ""
+    ? { input: { name: searchByTaskName } }
```

Comment:
Is this the same as?
`input: searchByTaskName ? { name: searchByTaskName } : {},`